aws/internal/keyvaluetags/service_tags_gen.go

```diff
@@ -1782,7 +1782,13 @@ func LightsailKeyValueTags(tags []*lightsail.Tag) KeyValueTags {
 	m := make(map[string]*string, len(tags))
 
 	for _, tag := range tags {
-		m[aws.StringValue(tag.Key)] = tag.Value
+		if tag.Value == nil {
```

Comment:
@stefansundin Thanks for this.
This source file is actually generated and will be recreated if `make gen` is run.
This behaviour may well be occurring for all relevant AWS services (I will do some spot tests) so should be fixed globally - changing

https://github.com/terraform-providers/terraform-provider-aws/blob/0286dba6e6f55c52863bdfa6662282a250a32a74/aws/internal/keyvaluetags/key_value_tags.go#L138

to

```go
 result[k] = aws.StringValue(v)
```

would be the more strategic fix.

Reply:
Ok. Updated. Let me know if I should make more changes.